configurations/leap_s05_pipeline.py

```diff
@@ -0,0 +1,229 @@
+from core_data_modules.cleaners import swahili
+from src.pipeline_configuration_spec import *
+
+NUM_EPISODES = 2
+
+def generate_rqa_coda_dataset_config(num_episodes=NUM_EPISODES):
```

Comment:
Why do we need this flexibility vs. always requiring the caller to set this or always using the constant?